pkgs/development/libraries/nyan/default.nix

```
@@ -0,0 +1,40 @@
+{ lib
+  , stdenv
+  , fetchFromGitHub
+  , clang_14
+  , cmake
+  , flex
+}:
+
+stdenv.mkDerivation {
+	pname = "nyan";
+	version = "0.2.0";
+
+	src = fetchFromGitHub {
+		owner = "SFTtech";
+		repo = "nyan";
+		rev = "7b5517f8c3c17b1fb2483f6dec882ccb84b70bbb";
+		sha256 = "sha256-9pTWXgbGNrvA6o90TDbouUJx9Q6xpSN+aGbnwXKm0V4=";
+	};
+
+	nativeBuildInputs = [
+		clang_14 # Just clang doesn't work on MACOS
+		cmake
+	];
+
+	buildInputs = [
+		flex
+	];
+
+	meta = with lib; {
+		description = "Data description language. A mixture of python, json, patch, wml, yaml and some new ideas";
```

Short `description` should be short, one sentence. There is no problem being a bit vague here:

```suggestion
		description = "A data description language";
```